radixdlt-cli/src/main/groovy/com/radixdlt/cli/SendTokens.groovy

```diff
@@ -0,0 +1,38 @@
+package com.radixdlt.cli
+
+import com.radixdlt.client.application.RadixApplicationAPI
+import com.radixdlt.client.atommodel.accounts.RadixAddress
+import picocli.CommandLine
+
+@CommandLine.Command(name = "send-token", mixinStandardHelpOptions = true,
+        description = "Send Token")
+class SendTokens implements Runnable {
+
+    @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
+    Composite.IdentityInfo identityInfo
+
+    @CommandLine.Option(names = ["-a", "--amount"], paramLabel = "AMOUNT", description = " Amount to send", required = true)
+    BigDecimal amount
+
+    @CommandLine.Option(names = ["-t", "--token"], paramLabel = "TOKEN", description = "Token to send", required = true)
+    String tokenString
+
+    @CommandLine.Option(names = ["-d", "--address"], paramLabel = "ADDRESS", description = "Address to which token is sent", required = true)
```

Comment:
`to`? or `recipient`?

Reply:
Will take off this class and add it back with right implement when we have need for this in command line